decentraland/unity-renderer
# feat: network flag now is visible in the wearables info card

#573 

Now the NFT info card shows the network the wearable belongs to. The implementation is not ideal, using the protocol in the `urn` and finding key words will work for now but wont last long. Unfortunately we cannot retrieve the information correctly until the catalysts provide it https://github.com/decentraland/catalyst/issues/535.

## To test it: 
Open the avatar editor and click in the small info button in the bottom left corner of a collectible. It should show the network properly.

Test L2 Collectible: https://play.decentraland.zone/index.html?renderer=urn:decentraland:off-chain:renderer-artifacts:feat/network_in_wearable_card&WITH_COLLECTIONS=urn:decentraland:mumbai:collections-v2:0xa5e32410f4d58f0b6c4d747287f1148f41e13fa3

Normal collections: https://play.decentraland.zone/index.html?renderer=urn:decentraland:off-chain:renderer-artifacts:feat/network_in_wearable_card

## Disclaimer
The user experience with the info card is not ideal, for example if you trigger it at the bottom of the screen, half of the card will be cut. Ignore that in the review.
